src/schema/objects/common_principles.yaml

```diff
@@ -6,19 +6,17 @@
 atlas:
   display_name: Atlas
   description: |
-    Knowledge about the brain, generally formalized with reference to a standard space (see the *Template* definition)
+    Knowledge about the brain, generally formalized with reference to a standard space (see the *Template* definition above)
     by means of spatiotemporal annotations such as landmarks, segmentations, parcellations, or probability maps.
+    One prominent manuscript regarding the specific aspects of atlases, such as their *regional resolution*,
+    is [Bijsterbosch et al. (2020)](https://doi.org/10.1038/s41593-020-00726-z).
 
-        The definition of atlas per Merriam-Webster is ‘a bound collection of maps (i.e. labeled brain regions
-        or quantitative aspects) and metadata (tables, or textual matter).
-        Within BIDS, atlases are broadly defined as a mapping between locations in a spatial coordinate systems
-        and descriptions associated with those locations.
-        Atlases are often built after *registering many subjects or maps into a space defined by a template*.
-        By analogy with geographical atlases, brain atlases can map brain locations to either discrete labels like a map
-        of countries does, or to continuous quantities like a topographic map does.
-
-        One prominent manuscript regarding the specific aspects of atlases, such as their *regional resolution*
-        is ([Bijsterbosch et al., 2020](https://doi.org/10.1038/s41593-020-00726-z)).
+        To further differentiate the concept of *atlas* and *template* throughout the specification, *template* will
+        designate a standardized stereotaxy frame where some feature or features are mapped through spatial normalization
+        and then possibly averaged.
+        Conversely, *atlas* will designate a specific methodology or algorithm to create meaningful partitions 
+        and other knowledge annotations about the brain such as landmarks, with reference to a
```

Comment:
Remove ", with reference to..." since it's not necessary here and is already in the template definition.

Reply:
Removed.